books/the_wealth_of_nations/license.md

```
@@ -0,0 +1,26 @@
+You are free to:
+
+- Share: Copy and redistribute the material in any medium or format.
+- Adapt: Remix, transform, and build upon the material.
+
+Under the following terms:
+
+- Attribution: You must give appropriate credit to Timeless Library (timelesslibrary.org), provide a link to the license, and indicate if changes were made.
+
+- NonCommercial: You may not use the material for commercial purposes (you cannot sell this book or include it in a paid product).
+
+To view a copy of this license, visit: http://creativecommons.org/licenses/by-nc/4.0/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)